blade-graphics/src/metal/resource.rs

```diff
@@ -198,6 +202,7 @@ impl crate::traits::ResourceDevice for super::Context {
             descriptor.set_array_length(desc.array_layer_count as u64);
             descriptor.set_mipmap_level_count(desc.mip_level_count as u64);
             descriptor.set_pixel_format(mtl_format);
+            descriptor.set_sample_count(desc.sample_count as _);
```

Comment:
I don't see where we are handling `ResolveTo` in Metal?

Reply:
in `metal/command.rs:223`,

```rust
let store_action = match rt.finish_op {
    crate::FinishOp::Store | crate::FinishOp::Ignore => {
        metal::MTLStoreAction::Store
    }
    crate::FinishOp::Discard => metal::MTLStoreAction::DontCare,
    crate::FinishOp::ResolveTo(ref view) => {
        at_descriptor.set_resolve_texture(Some(view.as_ref()));
        metal::MTLStoreAction::MultisampleResolve
    }
};
at_descriptor.set_store_action(store_action);
```